commons/connector/fitbit/fitbit_heart_rate_timeseries.avsc

```diff
@@ -0,0 +1,10 @@
+{
+  "namespace": "org.radarcns.connector.fitbit",
+  "type": "record",
+  "name": "FitbitHeartRateTimeSeries",
+  "doc": "Heart rate time-series data over a period of time from fitbit device.",
+  "fields": [
+    { "name": "datetime", "type": "string", "doc": "Local date of device (yyyy-MM-dd)." },
```

Comment:
please rename to `date`, and mention that this is a `ISO 8601 formatted local date (yyyy-MM-dd)`. And please add a `timeReceived` field as in other records.